modules/ci/commit0.module.yml

```diff
@@ -0,0 +1,24 @@
+module: 
+  title: "CI templates"
+  description: ""
+  author: ""
+  icon: ""
+  thumbnail: ""
+  readme: "" # defaults to README.md
+
+# Template variables to populate, these could be overwritten by the file spefic frontmatter variables
+templates: 
+  strictMode: true # will only parse files that includes the .tmpl.* extension, otherwise it will copy file 
+  delimiters:
+    - "<%"
+    - "%>"
+  destination: "{{.Metadata.Directory}}/{{.Metadata.Filename}}"
+
+# parameters required 
+# populates .Params variables
+params: 
+  - name: system
```

Comment:
What would this affect?

Reply:
so `params` would essentially be the form builder or CLI prompt configs. This is inspired by https://yeoman.io/authoring/user-interactions.html